CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.7.1 (Jun 2018)
```

Comment:
Maybe add links to corresponding PR's?

Reply:
We try to avoid this level of verboseness in CHANGELOG, just provide general descriptions of changes.